### PR TITLE
issue-79 Enhance TimeConverterTest to match best test practice

### PR DESCRIPTION
**Pull request by Laura-Royet**

### CODE REVIEW

File: server/container/util/src/test/java/org/apache/james/util/TimeConverterTest.java [hunk 1]
```diff
@@ -18,126 +18,223 @@
...
-	String unit = "msec";
-
-	assertEquals(TimeConverter.getMilliSeconds(AMOUNT, unit), time);
-	assertEquals(TimeConverter.getMilliSeconds(AMOUNT + " " + unit), time);
+    @Test 
+    public void getMilliSecondsShouldConvertValueWhenMsecUnit() {
+        //Given
+        long expected = 2;
+        //When
+        long actual = TimeConverter.getMilliSeconds(2,"msec");
```
Comment (mbaechler) [inline server/container/util/src/test/java/org/apache/james/util/TimeConverterTest.java]: sorry to be picky, but we usually put a space after a comma

**Comment by rouazana** [comment]
:+1:

**Comment by mbaechler** [comment]
:+1: only two fixups for this one, that's not bad for a start

**Comment by aduprat** [comment]
:+1:

**Comment by aduprat** [comment]
test this please

**Comment by Laura-Royet** [comment]
test this please

**Comment by Laura-Royet** [comment]
New PR:
https://github.com/linagora/james-project/pull/109

**Pull request closed without merging**